app/components/primer/beta/button.rb

```diff
@@ -81,7 +81,7 @@ class Button < Primer::Component
       renders_one :tooltip, lambda { |**system_arguments|
         raise ArgumentError, "Buttons with a tooltip must have a unique `id` set on the `Button`." if @id.blank? && !Rails.env.production?
 
-        ActiveSupport::Deprecation.warn("Buttons with visible text should not use a `label` tooltip. Consider using Primer::Beta::IconButton instead.") if system_arguments[:type] == :label
+        deprecation.warn("Buttons with visible text should not use a `label` tooltip. Consider using Primer::Beta::IconButton instead.") if system_arguments[:type] == :label
```

Comment:
Should this one be `::Primer::ViewComponents.`?

Reply:
You can call `deprecation` in anything that inherits from `Primer::Component` 👍